app/controllers/application_controller.rb

```diff
@@ -14,6 +15,36 @@ def shift_domain
     redirect_to request.url.gsub(host, 'iidx12.tk')
   end
 
+  def load_nav_routes
```

Comment:
__[rails_best_practices]__
* move model logic into model ( use_count > 4) [:ref](http://rails-bestpractices.com/posts/2010/07/21/move-model-logic-into-the-model/)
* move model logic into model (current_user use_count > 4) [:ref](http://rails-bestpractices.com/posts/2010/07/21/move-model-logic-into-the-model/)